Database/Corrections/cataQuestFixes.lua

```diff
@@ -4100,6 +4119,10 @@ function CataQuestFixes.Load()
         [26333] = { -- No Tanks!
             [questKeys.zoneOrSort] = zoneIDs.CHILL_BREEZE_VALLEY,
         },
+        [26334] = { -- Bloodlord Mandokir
+            [questKeys.preQuestSingle] = {},
+            [questKeys.preQuestGroup] = {26301,26332},
```

Comment:
Where did you find that information?

Reply:
Retested - just https://www.wowhead.com/cata/quest=26332/skullsplitter-mojo required